src/Microsoft.AspNetCore.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -365,6 +366,37 @@ public void Copy(Operation operation, object objectToApplyTo)
             }
         }
 
+        public void Test(Operation operation, object objectToApplyTo)
```

Comment:
Consider copying the relevant portions of the spec here like the others. At least they will all be wacky the same way.